config/provider.go

```diff
@@ -49,10 +50,8 @@ import (
 	"github.com/crossplane-contrib/provider-jet-azure/config/subnet"
 )
 
-const (
-	resourcePrefix = "azure"
-	modulePath     = "github.com/crossplane-contrib/provider-jet-azure"
```

Comment:
Looks like we still need this constant in line 136.

Reply:
added it back :)